packages/rxjs/spec/observables/fromEvent-spec.ts

```diff
@@ -352,7 +352,7 @@ describe('fromEvent', () => {
     };
 
     function selector(x: number, y: number, z: number) {
-      return [].slice.call(arguments);
+      return [].slice.call(x, y, z);
```

Comment:
This looks like an issue... it's not the same. Just leave the old code as is and add an ignore, I think.